3.7/ubuntu/Dockerfile

```diff
@@ -0,0 +1,138 @@
+# The official Canonical Ubuntu Bionic image is ideal from a security perspective,
+# especially for the enterprises that we, the RabbitMQ team, have to deal with
+FROM ubuntu:18.04 AS release
```

Comment:
I didn't understand from the issue linked above why multi-stage builds are a no-go. Everything works as expected locally (macOS 10.14.2, Docker Desktop 2.0.0.0-mac81), the final image is ~250MB currently, still working on the RabbitMQ part.

We haven't decided yet what will happen with Alpine-based variants. We are 100% behind Ubuntu Bionic, even at a company level, it makes sense to focus our efforts there. In other words, we are happy to put the proverbial stamp on the Bionic-based RabbitMQ images. Those that value this endorsement over image size will have an easy time choosing 😉

Reply:
> I didn't understand from the issue linked above why multi-stage builds are a no-go. 

That'd be because we don't `docker build` the images here directly (except for testing) -- those go through the Docker standard "Official Images" process outlined in https://github.com/docker-library/official-images, and build on that infrastructure via a separate tool which doesn't support multi-stage builds and isn't trivial to update _to_ support multi-stage builds.

I'm happy to help convert this into a non-multi-stage build (and still have it clean up, just within one layer so the cleanup actually takes effect) once you get something you're happy with, though. :+1:

> Those that value this endorsement over image size will have an easy time choosing :wink:

:metal: SGTM

Reply:
This is now clear, thank you!